app/controllers/administrative-units/index.js

```diff
@@ -27,6 +27,7 @@ export default class AdministrativeUnitsIndexController extends Controller {
   @tracked municipality = '';
   @tracked province = '';
   @tracked classificationId = '';
+  @tracked selections = '';
```

Comment:
The name is a bit vague, wouldn't `classificationIds` work ? Or `selectedClassificationIds`, something that shows it's about classifications haha